keps/sig-network/1672-tracking-terminating-endpoints/README.md

```diff
@@ -148,9 +155,123 @@ of the [EndpointSlice API work](/keps/sig-network/20190603-endpointslices/README
 Since this is an addition to the EndpointSlice API, the version skew strategy will follow that
 of the [EndpointSlice API work](/keps/sig-network/20190603-endpointslices/README.md).
 
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+###### How can this feature be enabled / disabled in a live cluster?
+
+- [X] Feature gate (also fill in values in `kep.yaml`)
+  - Feature gate name: EndpointSliceTerminatingCondition
+  - Components depending on the feature gate: kube-apiserver and kube-controller-manager
+
+###### Does enabling the feature change any default behavior?
+
+Yes, terminating endpoints are now included as part of EndpointSlice API. The `ready` condition of an EndpointSlice will always be `false` to ensure consumers do not send traffic to terminating endpoints unless new conditions are checked.
+
+###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
+
+Yes. On rollback, terminating endpoints will no longer be included in EndpointSlice and the `terminating` and `serving` conditions will not be set.
```

Comment:
1. The will get removed on the nearest EndpointSlice sync, right?
[i.e. they won't get removed automaticaly]

2. What is `serving` condition? It hasn't been defined so far in this kep. Was this introduced by another KEP?

Reply:
It was introduced last minute during implementation because we can't change the existing semantics around `ready`. I'll add more on this

Reply:
added words on the new `serving` condition we added

Reply:
Historically "ready" means "pod ready and not terminating".  "Ready and terminating" is a thing we care about, but we can't break the historical meaning of "ready" (or clients who consider all "ready" endpoints would suddenly include "ready and terminating".  So we added 2 new fields that are properly orthogonal.